src/styles/_catalog.scss

```diff
@@ -1,9 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 487px) {
+    grid-template-columns: 200px;
+  }
+  @media (min-width: 488px) and (max-width: 767px) {
+    grid-template-columns: 200px 200px;
+  }
+  @media (min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
You can make a loop to create all media queries. Just create a correctly mapped list to iterate.

Reply:
It only will be shortened with previously suggested avoiding max-width I think

Reply:
got it - have used it - following the best practices